app/models/game.rb

```diff
@@ -49,4 +49,16 @@ def get_piece(x, y)
     self.pieces.where(x_position: x, y_position: y).take
   end
 
+  def set_default_turn
```

Comment:
This should probably be a bang method.
https://stackoverflow.com/questions/612189/why-are-exclamation-marks-used-in-ruby-methods